src/components/advanced-color-picker/advanced-color-picker.spec.js

```diff
@@ -0,0 +1,390 @@
+import React from 'react';
+import EnzymeToJson from 'enzyme-to-json';
+import { mount as enzymeMount } from 'enzyme';
+import { act } from 'react-test-renderer';
+import 'jest-styled-components';
```

Comment:
This can be removed as we import it globally as well

Reply:
fixed